fabric-textures-v0/src/main/java/net/fabricmc/fabric/api/client/texture/ColorSniffer.java

```diff
@@ -0,0 +1,52 @@
+package net.fabricmc.fabric.impl.client.texture;
+
+import net.fabricmc.fabric.mixin.client.texture.SpriteAccessor;
+import net.minecraft.client.texture.NativeImage;
+import net.minecraft.client.texture.Sprite;
+
+/**
+ * Utility class for averaging Sprite colors.
+ * You should call one of these methods!
+ */
+public class ColorSniffer {
```

Comment:
Fix the license and make the class final

Reply:
Also make a private constructor